src/deepmol/datasets/datasets.py

```diff
@@ -617,8 +629,7 @@ def save_to_csv(self, path):
 
         df.to_csv(path, index=False)
 
-    # TODO: test load and save
-    def load_features(self, path, sep=',', header=0):
+    def load_features(self, path: str, sep: str = ',', header: int = 0):
```

Comment:
Personal taste also: I prefer **load**, as it is less verbose. Also, shouldn't it be better to add ****kwargs** instead of defining the pandas-specific parameters such as sep and header? That way, any parameter of the **read_csv** pandas method could be passed to this method.

Reply:
Hum, in this case, it only loads features (the `X`) so I don't think I agree. We have `save_features` also. Agree with the **kwargs.

Reply:
Ok, sounds fair